### ISSUE TITLE
[Bug]: Skip button missing on Tizen TV but works on Windows

### ISSUE THREAD

**Issue by Jacob-077** [intro-skipper/intro-skipper#383]
### Self service debugging

- [X] I use Jellyfin 10.10.2 (or newer) and my permissions are correct

### Describe the bug

I installed Jellyfin for Tizen with intros skipping (Jellyfin-10.10.z-intros.wgt) from: https://github.com/jeppevinkel/jellyfin-tizen-builds

On Windows, the skip button appears as expected, but on the Tizen TV, it does not. However, if I enable auto-skip, it works.

Since the plugin is functioning correctly, how can I get the skip button to appear on the TV?

### Jellyfin install method

Sideloading

### Container image/tag or Jellyfin version

10.10

### Operating System

Window 10

### IMDb ID of that TV Series

tt2304589

### Support Bundle

```shell
* Jellyfin version: 10.10.3
* Plugin version: 1.10.10+f275e188da52
* Queue contents: 1396 episodes, 167 seasons
* Warnings: `None`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 7.0.2-Jellyfin Copyright (c) 2000-2024 the FFmpeg developers
built with clang version 19.1.3
configuration: --cc=clang --pkg-config-flags=--static --extra-cflags=-I/clang64/ffbuild/include --extra-ldflags=-L/clang64/ffbuild/lib --prefix=/clang64/ffbuild/jellyfin-ffmpeg --extra-version=Jellyfin --disable-ffplay --disable-debug --disable-doc --disable-sdl2 --disable-ptx-compression --enable-lto=thin --enable-gpl --enable-version3 --enable-schannel --enable-iconv --enable-libxml2 --enable-zlib --enable-lzma --enable-gmp --enable-chromaprint --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libharfbuzz --enable-libass --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libwebp --enable-libvpx --enable-libzimg --enable-libx264 --enable-libx265 --enable-libsvtav1 --enable-libdav1d --enable-libfdk-aac --enable-opencl --enable-dxva2 --enable-d3d11va --enable-d3d12va --enable-amf --enable-libvpl --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      59.  8.100 / 59.  8.100
libavcodec     61.  3.100 / 61.  3.100
libavformat    61.  1.100 / 61.  1.100
libavdevice    61.  1.100 / 61.  1.100
libavfilter    10.  1.100 / 10.  1.100
libswscale      8.  1.100 /  8.  1.100
libswresample   5.  1.100 /  5.  1.100
libpostproc    58.  1.100 / 58.  1.100
```
```


### Jellyfin logs

_No response_

**Comment by AbandonedCart** [comment]
I believe you are waiting on the release of a recently merged pull request from jellyfin. 

It’s been merged into their preview branch in the last two days, so the best thing to do for now is be patient.

**Comment by jumoog** [comment]
I'll close it as it's not related to our plugin.

**Comment by jumoog** [comment]
Also see: https://github.com/jellyfin/jellyfin-tizen/issues/305

**Comment by morpheus133** [comment]
Currently there is a build for Tizen what inject this:
https://raw.githubusercontent.com/intro-skipper/intro-skipper/refs/heads/10.10/IntroSkipper/Configuration/inject.js
to the index.html.
It was prevoiusly working, is this till needed, or with the new modification it will not?
thanks.
